spec/attribute.dd

```diff
@@ -328,9 +328,11 @@ $(GNAME DeprecatedAttribute):
         import std.format;
         enum Message = format("%s and all its members are obsolete", Foobar.stringof);
         deprecated(Message) class Foobar {}
-        auto f = new Foobar();   // Deprecated: class test.Foobar is deprecated - Foobar and all its members are obsolete
+        auto f = new Foobar();   // Deprecated: class test.Foobar is deprecated - Foobar
+             // and all its members are obsolete
         deprecated(format("%s is also obsolete", "This class")) class BarFoo {}
-        auto bf = new BarFoo();  // Deprecated: class test.BarFoo is deprecated - This class is also obsolete
+        auto bf = new BarFoo();  // Deprecated: class test.BarFoo is deprecated - This
+             // class is also obsolete
```

Comment:
Looks good:

![image](https://user-images.githubusercontent.com/4370550/33920661-63591aaa-dfbf-11e7-8e81-21b2fb1bc3f3.png)

Reply:
And we have enough space on dlang.org too:

![image](https://user-images.githubusercontent.com/4370550/33920740-d9743f12-dfbf-11e7-948f-38e20404b1f3.png)